src/odemis/driver/semnidaq.py

```diff
@@ -2292,6 +2292,7 @@ def __init__(self, name: str, role: str, parent: AnalogSEM,
         pixel of the frame, and goes down at the end of the last pixel.
         For each of the 3 types of TTL, a dictionary can be provided with the following keys:
         * "port": list of int, defining the digital output port(s) to use.
+        * "inverted": list of boolean, defining for each port whether the inactive state is low (False), or high (True).
         * "affects": list of str, containing the names of components which are physically connected
         to the signal.
         :param scanning_ttl (None or dict of int -> (bool, bool, Optional[str])):
```

Comment:
can you swap scanning_ttl and image_ttl in the docstring to match the order of the arguments in the definition?